src/server/helpers/token.js

```diff
@@ -6,7 +6,7 @@ import response from './errorHandler';
 dotenv.config();
 
 const { SECRET } = process.env;
-const tokenLife = '15m';
+const tokenLife = '5h';
```

Comment:
Are you sure this is the secure life time for token?

Reply:
changed to 30m